distribution/test/itests/test-itests-alliance/src/test/java/org/codice/alliance/test/itests/VideoTest.java

```diff
@@ -117,6 +117,8 @@ public void testStreamingVideo() throws Exception {
                 false,
                 null);
 
+        Thread.sleep(2000);
```

Comment:
I put in to make sure the stream end is detected by timeout and calculates the final update to the parent metacard